frontend/src/components/Card.tsx

```diff
@@ -58,6 +58,14 @@ const Card = ({
     setShowAll(!toggleLanguagesAndTopics)
   }
 
+  const handleHeadingColor = (e, isMouseEnter) => {
```

Comment:
I like all of these changes except for this one. I think having the links with different color makes it too colorful and doesn't match our overall style for the platform.

Can we revert this particular change and keep the rest?